src/main/java/test/OlegLoginTest.java

```diff
@@ -0,0 +1,21 @@
+package main.java.test;
+
+import static org.testng.Assert.*;
+import main.java.pages.header.UnlogUserInfo;
+import main.java.pages.login.LoginForm;
+import main.java.tools.WebDriver;
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class OlegLoginTest {
+	@Test
+	public void testLoginForm(){
+		WebDriver webDriver = new WebDriver ();
+		UnlogUserInfo unlogUserInfo = new UnlogUserInfo ();
+		unlogUserInfo.clickSignIn ();
+		LoginForm loginForm = new LoginForm (  );
+		String expected = "Admin admin";
+		String actual = WebDriver.driver.findElement ( By.cssSelector ( "#_desktop_user_info > div > a.account > span" ) ).getText ();
+		assertEquals ( expected, actual );
```

Comment:
you know that in testng it's vice versa

https://static.javadoc.io/org.testng/testng/6.8.17/org/testng/Assert.html#assertEquals(boolean,%20boolean)

Reply:
oh yes as i said i just test it, it will be correct soon)